Pod/Classes/Format/CardNumberFormatter.swift

```diff
@@ -36,8 +36,8 @@ public final class CardNumberFormatter {
      
      - returns: The unformatted card number string representation.
      */
-    public func unformattedCardNumber(cardNumberString: String) -> String {
-        return cardNumberString.stringByReplacingOccurrencesOfString(self.separator, withString: "")
+    public func unformattedCardNumber(_ cardNumberString: String) -> String {
```

Comment:
`unformat(cardNumberString: String)` might work better for the Swift API rule ["Prefer method and function names that make use sites form grammatical English phrases."](https://swift.org/documentation/api-design-guidelines/#strive-for-fluent-usage)